src/common/persistence/timescaledb/entities/sum-daily.entity.ts

```diff
@@ -0,0 +1,32 @@
+import { PrimaryColumn, ViewColumn, ViewEntity } from 'typeorm';
+
+@ViewEntity({
+  expression: `
+    SELECT
+      time_bucket('1 day', timestamp) AS time, series, key,
+      last(value, timestamp) AS last,sum(value) AS sum
```

Comment:
I don't think you would need the last value for a volume metric. Only sum over a time bucket would be relevant